src/Api.php

```diff
@@ -269,24 +279,9 @@ public function validatePurgeCredentials() {
    *   FALSE if purge failed, TRUE is successful.
    *   */
   public function purgeAll() {
```

Comment:
Wondering if we still want to preserve the ability to call `/purge_all`? We can pass a param into `purgeAll()` that defaults to `FALSE` that callers can change if they really want to purge all.

Reply:
If we are going to use site IDs then yes we should make it a thing of last resort.

Reply:
There might be exceptional circumstances in which you want to purge everything in the entire service. The only example I can think of is static files (e.g. CSS, JS, PNG etc), to which Nginx (or Apache) serves, these files get no cache tags, but they do have a 2 week cache lifetime.

As a last resort being able to nuke the entire cache might be useful. As long as there is appropriate messaging to inform the user that this should be only be done in exceptional circumstances etc.

Reply:
Indeed I would leave purgeAll as the purge of last resort.

Reply:
So we can then have like two actions. 
We should rename current purge like it is right now from "Purge / invalidate all content" to "Purge / invalidate all site content"
The other one we call "Purge / invalidate whole service" for purging entire service.